app/app.go

```diff
@@ -262,12 +257,12 @@ func NewWasmApp(
 
 	// Create IBC Keeper
 	app.ibcKeeper = ibc.NewKeeper(
-		app.cdc, keys[ibc.StoreKey], app.stakingKeeper, scopedIBCKeeper,
+		app.cdc, appCodec, keys[ibc.StoreKey], app.stakingKeeper, scopedIBCKeeper,
```

Comment:
I am getting confused.

Why does this need both `*std.Codec` and `*codec.Codec` and what are these for?

(Not your issue, but if you can give me a 1 paragraph issue, I would be grateful)

Reply:
I have not looked into details but migration to protobuf is not done for this module